suite-native/app/package.json

```diff
@@ -4,7 +4,7 @@
     "suiteNativeVersion": "24.5.0",
     "main": "index.js",
     "scripts": {
-        "android": "expo run:android",
+        "android": "expo run:android && yarn reverse-ports",
```

Comment:
This won't work since `yarn android` doesn't finish until app is killed. Just leave the script as it is to be called manually in a new window, still a step forward..

Reply:
fixed